samples/tiled-convolution/CMakeLists.txt

```diff
@@ -1,4 +1,28 @@
 set(SOURCE_NAME "tiled-convolution")
+
```

Comment:
I think, given all these comments, I'd prefer that instead of all these changes, we unconditionally do what @mmha suggests:
```CMake
target_compile_definitions(
  ${SOURCE_NAME} PRIVATE LOCAL_SIZE_M=1 LOCAL_SIZE_N=32)
```
We can the complexity later - but I think YAGNI